### ISSUE TITLE
[BUG] Duplicate type declaration in pstream.d.ts

### ISSUE THREAD

**Issue by vicrep** [twilio/twilio-client.js#143]
- [x] I have verified that the issue occurs with the latest twilio.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-client.js/blob/master/CHANGELOG.md).
 - [x] I reviewed the [Common Issues](https://github.com/twilio/twilio-client.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [x] I verified that the [Quickstart](https://github.com/TwilioDevEd/client-quickstart-js) application works in my environment.
 - [x] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Code to reproduce the issue:**

This is a new error occurring as of 1.10.1, on a project running typescript 3.7.5. This is similar to #139 but making a new ticket as it's specific to 1.10.1, and said ticket seemed to refer to an older version.

Run TSC with `skipLibCheck` disabled on a project requiring twilio-client:

```
node_modules/twilio-client/es5/twilio/pstream.d.ts:1:12 - error TS2300: Duplicate identifier 'PStream'.

1 export var PStream: typeof PStream;
             ~~~~~~~

node_modules/twilio-client/es5/twilio/pstream.d.ts:17:18 - error TS2300: Duplicate identifier 'PStream'.

17 declare function PStream(token: string, uri: string, options?: any): PStream;
                    ~~~~~~~


Found 2 errors.
```

 

**Expected behavior:**

Should pass type checking :)

**Software versions:**

 - [ ] Browser(s): N/A
 - [ ] Operating System: N/A
 - [x] twilio.js: 1.10.1
 - [x] Third-party libraries (e.g., Angular, React, etc.): typescript 3.7.5

**Comment by charliesantos** [comment]
Hi @vicrep , thank you for submitting. We will investigate further.
Is this issue only happening with Angular build, similar to the ticket you referenced?

**Comment by charliesantos** [comment]
Jira [CLIENT-7544](https://issues.corp.twilio.com/browse/CLIENT-7544)

**Comment by vicrep** [comment]
Hi @charliesantos , thanks for the quick response!

This happens on a react app using typescript, and the type error seems specific to the internal types of the library, not interop with angular.

**Comment by charliesantos** [comment]
Thank you for this information @vicrep .

**Comment by charliesantos** [comment]
Fixed in #144 and will be included in 1.11. Please watch out for the release.

**Comment by vicrep** [comment]
Great! Thanks for the quick turnaround @charliesantos , appreciate it :)
